### ISSUE TITLE
cublas runtime error : the GPU program failed to execute at

### ISSUE THREAD

**Issue by hfutxrg** [NVIDIA/DIGITS#1242]
Hi I installed digits and torch on Ubuntu 14.04 following the steps in http://www.nvidia.com/object/torch-library-installation.html and https://www.pugetsystems.com/labs/hpc/NVIDIA-DIGITS-Install-783/ 

But when I train mnist with Torch, it has following error:
ERROR: /usr/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] /usr/share/lua/5.1/nn/Container.lua:67:

The log file has more specific error message:
2016-11-03 15:19:05 [INFO ] started training the model
2016-11-03 15:20:24 [INFO ] Validation (epoch 0): loss = 2.2997478747419, accuracy = 0.10681424189892
2016-11-03 15:20:24 [FAIL] /usr/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] /usr/share/lua/5.1/nn/Container.lua:67:
In 7 module of nn.Sequential:
**/usr/share/lua/5.1/nn/Linear.lua:99: cublas runtime error : the GPU program failed to execute at /tmp/buildd/torch7-0.9.99/extra/cutorch/lib/THC/THCBlas.cu:211**
stack traceback:
[C]: in function 'addmm'
/usr/share/lua/5.1/nn/Linear.lua:99: in function 'accGradParameters'
/usr/share/lua/5.1/nn/Module.lua:32: in function </usr/share/lua/5.1/nn/Module.lua:29>
[C]: in function 'xpcall'
/usr/share/lua/5.1/nn/Container.lua:63: in function 'rethrowErrors'
/usr/share/lua/5.1/nn/Sequential.lua:84: in function </usr/share/lua/5.1/nn/Sequential.lua:78>
[C]: in function 'xpcall'
/usr/share/lua/5.1/threads/threads.lua:234: in function 'callback'
/usr/share/lua/5.1/threads/queue.lua:65: in function </usr/share/lua/5.1/threads/queue.lua:41>
[C]: in function 'pcall'
/usr/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
WARNING: If you see a stack trace below, it doesn't point to the place where this error occured. Please use only the one above.

The CUDA version is 7.5 and cublas library is also installed. DIGITS version is 4.0. I didn't find the file /tmp/buildd/torch7-0.9.99/extra/cutorch/lib/THC/THCBlas.cu.  Also I don't know why cublas runtime error happened here. Any help is appreciated.

I just found that I have the same problem when using Caffe framework:
F1103 17:07:29.459863 14161 math_functions.cu:27] Check failed: status == CUBLAS_STATUS_SUCCESS (13 vs. 0)  CUBLAS_STATUS_EXECUTION_FAILED
*** Check failure stack trace: ***
F1103 17:07:29.459875 14081 math_functions.cu:27] Check failed: status == CUBLAS_STATUS_SUCCESS (13 vs. 0)  CUBLAS_STATUS_EXECUTION_FAILEDF1103 17:07:29.459969 14162 math_functions.cu:27] Check failed: status == CUBLAS_STATUS_SUCCESS (13 vs. 0)  CUBLAS_STATUS_EXECUTION_FAILEDF1103 17:07:29.460074 14160 math_functions.cu:27] Check failed: status == CUBLAS_STATUS_SUCCESS (13 vs. 0)  CUBLAS_STATUS_EXECUTION_FAILED

The output of "ldconfig -p | grep libcublas" is:
libcublas.so.7.5 (libc6,x86-64) => /usr/local/cuda-7.5/targets/x86_64-linux/lib/libcublas.so.7.5
libcublas.so (libc6,x86-64) => /usr/local/cuda-7.5/targets/x86_64-linux/lib/libcublas.so

So DIGITS 4 cannot use the libcublas in cuda 7.5? 

Regards,
Rengan

**Comment by gheinrich** [comment]
Hello, do you have a Pascal GPU (in which case you want to upgrade to CUDA 8.0)?

**Comment by gheinrich** [comment]
Close due to inactivity.
